static/docs/user-guide/install/dev.md

```diff
@@ -2,14 +2,13 @@
 
 If you want to test the latest version of DVC, before the official release, you
 can install it from the code on GitHub. In this case it is highly recommended to
-use [virtualenv](https://virtualenv.pypa.io/en/stable/), in order to encapsulate
-your testing environment.
+use [virtualenv](https://virtualenv.pypa.io/en/stable/) in order to encapsulate
+your testing environment, otherwise it will automatically upgrade your system's
```

Comment:
it depends on how DVC is installed, so this is not correct. Also, the whole point of using the preview version usually is to use it as a replacement, not along with an old one. It's the case when early adopters what the most recent shiny features.